### PR TITLE
chore: First version

### PR DESCRIPTION
**Pull request by maciejmacq-dev**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-    name  = local.subresource_name_from_descriptor
-    input = var.sub_resource.example_var
+module "terraform_gke_helm_release" {
+  context                    = module.this.context
+  source                     = "git::https://github.com/getindata/terraform-gke-helm-release.git?ref=v1.1.1"
+  kubernetes_namespace       = var.kubernetes_namespace
+  create_namespace           = true
+  project_id                 = var.project_id
+  name                       = var.app.name
+  service_account_value_path = "serviceAccount.name"
```
Comment (mariusz89016) [inline main.tf:18]: does that work with `null` value as described here? https://github.com/runatlantis/helm-charts/blob/main/charts/atlantis/values.yaml#L463

File: examples/simple/Makefile [hunk 2]
```diff
@@ -2,10 +2,10 @@ init:
 	terraform init
 
 plan:
-	terraform plan -out tfplan
+	terraform plan -var-file fixtures.tfvars -out tfplan
```
[the remark this answers is not in the document]
Comment (maciejmacq-dev) [reply examples/simple/Makefile:5]: fixex

File: examples/complete/README.md [hunk 3]
```diff
@@ -1,20 +1,62 @@
 # Complete Example
+Webhook configuration
+https://www.runatlantis.io/docs/deployment.html#kubernetes-helm-chart
```
[the remark this answers is not in the document]
Comment (maciejmacq-dev) [reply examples/complete/README.md:3]: true, fixed

File: main.tf [hunk 4]
```diff
@@ -1,36 +1,34 @@
...
 
-resource "null_resource" "subresource" {
-  count = local.enabled ? 1 : 0
-
-  triggers = {
-    name  = local.subresource_name_from_descriptor
-    input = var.sub_resource.example_var
+module "terraform_gke_helm_release" {
+  context                    = module.this.context
+  source                     = "git::https://github.com/getindata/terraform-gke-helm-release.git?ref=v1.1.1"
```
[the remark this answers is not in the document]
Comment (maciejmacq-dev) [reply main.tf:13]: yes,we can use this example  Fixed :)

File: main.tf [hunk 5]
```diff
@@ -1,36 +1,34 @@
...
+
+  descriptor_formats = {
+    gcp-service-account = {
+      labels = ["namespace", "environment", "name"]
+      format = "sa-%v-%v-%v"
+    }
   }
+  values = concat([templatefile("${path.module}/template/values.yaml.tftpl", { repoConfig = indent(2, module.atlantis_repo_config.repos_config_yaml) })], var.values)
+
+  roles = ["roles/compute.admin"]
```
[the remark this answers is not in the document]
Comment (maciejmacq-dev) [reply main.tf:28]: i think we have to change roles = ["roles/compute.admin"] to roles = var.role and add
variable "roles" {
  description = "Service Account roles"
  type        = list(string)
  default     = []
}
And then in example user can specify needed roles WDYT

[the remark this answers is not in the document]
Comment (maciejmacq-dev) [reply main.tf:28]: ok we decided to add roles = ["roles/editor"]

**Pull request merged**